src/UnityExtension/Assets/Editor/GitHub.Unity/UI/BranchesView.cs

```diff
@@ -134,19 +134,23 @@ private void Render()
 
         private void BuildTree()
         {
+            if (treeLocals == null)
+            {
+                treeLocals = new Tree();
+                treeLocals.ActiveNodeIcon = Styles.ActiveBranchIcon;
+                treeLocals.NodeIcon = Styles.BranchIcon;
+                treeLocals.FolderIcon = Styles.FolderIcon;
+
+                treeRemotes = new Tree();
```

Comment:
treeLocals and treeRemotes are always either defined or null together right? It isn't worth having a separate if for treeRemotes?

Reply:
They always get instantiated together, so I figured I could save the `boolean` check.